PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/forms/AddNewClusterForm.java

```diff
@@ -1,5 +1,7 @@
 package com.microsoft.azuretools.azureexplorer.forms;
 
+import com.microsoft.azuretools.telemetry.AppInsightsClient;
+import com.microsoft.intellij.hdinsight.messages.HDInsightBundle;
```

Comment:
Maybe we could put all telemetry related constants in `hdinsght-node-common` project for both intellij and eclipse.